api/swagger/swagger.yaml

```diff
@@ -205,7 +218,11 @@ paths:
         "200":
           description: Success
           schema:
-            $ref: "#/definitions/Event"
+            properties:
```

Comment:
I pulled the code and it gives me such error when I tried to get an event by id:
**{"message":"Response validation failed: failed schema validation","code":"SCHEMA_VALIDATION_FAILED","failedValidation":true,"results":{"errors":[{"code":"INVALID_TYPE","message":"Expected type object but found type string","path":["events","0","author"]}],"warnings":[]},"originalResponse":{"type":"Buffer","data":[123,34,101,118,101,110,116,115,34,58,91,123,34,95,105,100,34,58,34,53,97,52,54,55,56,100,102,49,57,48,48,**

I think there might be something wrong with the Author field. 
I spent a few hours trying to fix it but didn't get any luck. 
Could you please take a look when you have time?
Thanks.